### PR TITLE
Update file structure and fix extract in dispatcher

### PR DESCRIPTION
**Pull request by EstebanOlmedo**
This changes the structure of files in order to divide operations in images from the feature extraction. Also fix the extract operation in dispatcher to return the actual values instead of the dummy ones.

### CODE REVIEW

File: backend/dispatcher.py [hunk 1]
```diff
@@ -12,14 +13,12 @@ def verify_image_content(img, id):
         exit(-1)
 
 def compare(id1, id2):
```
Comment (DanielMontesGuerrero) [inline backend/dispatcher.py]: beforeId, afterId are easier to understand tha id1, id2

File: backend/image_processing/border_detection/border_detection.py
Comment (DanielMontesGuerrero) [inline backend/image_processing/border_detection/border_detection.py:1]: Why move this file to a folder with the same name?

Comment (EstebanOlmedo) [reply backend/image_processing/border_detection/border_detection.py:1]: Do we want to keep this file? I think we don't use it anymore.

Comment (DanielMontesGuerrero) [reply backend/image_processing/border_detection/border_detection.py:1]: then we can delete it

**Review by DanielMontesGuerrero — APPROVED** [review]

**Pull request merged**